### PR TITLE
GEN-1457 | Move partner service to widget feature

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Remove unused partner service and init endpoint

- Move relevant logic from service into widget feature

- Remove widget events from regular checkout

## Justify why they are needed

- I removed the code related to parsing search params (pre-fill etc.), however, I plan to reintroduce this when we develop that feature in the widget.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3544** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3544?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3545** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3545?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3546** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3546?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #3547** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3547?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #3548** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3548?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #3549** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3549?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #3550** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3550?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #3551** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3551?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                  * **PR #3558** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3558?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3558?utm_source=stack-comment).

**Review by guilhermespopolin — APPROVED** [review]
[shippingtime](https://app.graphite.dev/macros/HedvigInsurance)

File: apps/store/src/pages/debugger/iframe.tsx
Comment (gustaveen) [inline apps/store/src/pages/debugger/iframe.tsx:1]: I had missed that, we had a iframe debugger page, neat! 🙌

**Review by gustaveen — APPROVED** [review]
So fresh and so clean! 🧹

**Comment by robinandeer** [comment]
### Merge activity

* **Nov 14, 4:59 PM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3558) rebased this pull request as part of a merge.

**Pull request merged**